styles.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,700,900');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+  margin-left: 0;
+  width: 23.4375rem;
+  height: 5.75rem;
+}
```

Comment:
- [ ] You've done a good job aligning the background image but I noticed that there are some new design issues created as a result of the change specifically the vertical spacing in the toolbar seems to be a bit too much and this is mainly because of the Fixed height property you've given it at line `23` and the `margin-top` at line `18` and `28` and this causing a discrepancy when compared with the Figma design. As a front-end developer, you have to pay attention to small details like this as they make for a polished UI.
- Your design:
  ![image](https://github.com/umarahgul/Portfolio/assets/104968236/1537b912-e2e7-444d-8a10-922c32a27501)
- Figma design:
  ![firefox_4lJC35EiWX](https://github.com/umarahgul/Portfolio/assets/104968236/a01db147-9de7-47ce-bff6-46a7e4220330)

Reply:
@AbelG101 . Thanks very much for reviewing my code. I am glad you pointed out the discrepancy but can you explain if I am getting you right? you need me to reduce that space at the top?